htdocs_symfony/src/Entity/CachesEntity.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Oc\Entity;
+
+use Oc\Repository\AbstractEntity;
+#use Symfony\Component\Security\Core\User\UserInterface; ??
+
+#class CachesEntity extends AbstractEntity implements UserInterface
```

Comment:
If not in use - than remove ;)

Reply:
Maybe it will be needed in the future? ;)
If there is a User controller and a Caches controller, why should there be no Settings controller?